### PR TITLE
OSSMDOC-118

### PR DESCRIPTION
**Pull request by neal-timpe**
https://issues.redhat.com/browse/OSSMDOC-118

Kiail link moved from the application launcher.

### CODE REVIEW

File: modules/ossm-observability-access.adoc [hunk 1]
```diff
@@ -6,13 +6,15 @@
...
 = Accessing the Kiali console
 
-To access the console, in the menu bar, click the *Application launcher* > *Kiali*.
+To access the console, you must have the Kiali operator installed and a deployed an application in a project.
 
-1. In the {product-title} menu bar, click the *Application launcher* > *Kiali*.
+1. Use the perspective switcher to switch to the Developer perspective. 
 
-2. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console.
+2. Click *Topology*.
```
Comment (jshaughn) [inline modules/ossm-observability-access.adoc:13]: For topology, eligible topology nodes will offer a link.  For Admin, eligible projects will have link.

File: modules/ossm-observability-access.adoc [hunk 2]
```diff
@@ -6,13 +6,15 @@
...
 
-3. Select the project for your service in the *Namespace* field. If you have installed the Bookinfo example, select `bookinfo`.
+3. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console.
+
+4. Click `Kiali` from the toolbar.
```
Comment (jshaughn) [inline modules/ossm-observability-access.adoc:17]: Not sure which toolbar this refers to.  After step 3 you should be in the Kiali console.

**Review by jshaughn — CHANGES_REQUESTED** [review]

**Comment by neal-timpe** [comment]
@jshaughn Can you take another look at this?

File: modules/ossm-observability-access.adoc [hunk 3]
```diff
@@ -6,13 +6,27 @@
...
+
+4. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console.
+
+.For developers
+
+1. Use the perspective switcher to switch to the Developer perspective. 
+
+2. Click *Topology*.
+
+3. Click `Kiali` from the toolbar.
```
Comment (jshaughn) [inline modules/ossm-observability-access.adoc:27]: From the topology view the links are not from the toolbar, the links are from relevant nodes in the topology (i.e. a node in a project registered with service mesh).

**Review by jshaughn — CHANGES_REQUESTED** [review]
All good with one exception, see the comment.

**Comment by neal-timpe** [comment]
@jshaughn One more time. Can you approve if this is right?

**Review by jshaughn — APPROVED** [review]
Looks good.

File: modules/ossm-observability-access.adoc [hunk 4]
```diff
@@ -6,13 +6,27 @@
...
+
+4. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console.
+
+.For developers
+
+1. Use the perspective switcher to switch to the Developer perspective. 
+
+2. Click *Topology*.
+
+3. Click `Kiali` from a node in the topology you have configured for the service mesh.
```
Comment (hhovsepy) [inline modules/ossm-observability-access.adoc:27]: Kiali link is not in each topology node but only one on toolbar on the right side.
Screenshot from 2020-12-11 10-43-27

File: modules/ossm-observability-access.adoc [hunk 5]
```diff
@@ -6,13 +6,27 @@
...
-1. In the {product-title} menu bar, click the *Application launcher* > *Kiali*.
+.For administrators
 
-2. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console.
+1. Use the perspective switcher to switch to the Administrator perspective. 
 
-3. Select the project for your service in the *Namespace* field. If you have installed the Bookinfo example, select `bookinfo`.
+2. Click *Home* > *Projects*.
+
+3. In the Launcher section, click `Open on Kiali`.
```
Comment (hhovsepy) [inline modules/ossm-observability-access.adoc:17]: Here 'Launcher' sections are 2, one in Details tab, another in Overview tab of project details page.

**Review by mattmahoneyrh — COMMENTED** [review]
Seems the Kiali link in the project Overview.Launcher needs to be added

**Comment by neal-timpe** [comment]
@mattmahoneyrh It looks to me like it's in there in the administrators section.

**Comment by neal-timpe** [comment]
@hhovsepy @mattmahoneyrh I made a change to the administrator perspective. I understand it's in overview and details in projects, but I think we should only include one in the docs.

**Review by hhovsepy — APPROVED** [review]
looks good now

**Review by mattmahoneyrh — APPROVED** [review]

**Review by JStickler — CHANGES_REQUESTED** [review]
My only comment is that very last line (GitHub won't let me leave a comment there).  Since we're talking about two different consoles in the same topic, I think it would help to be as specific as possible which console we're talking about.

"When you first log in [to the Kiali Console],  you see the Overview page which displays all the namespaces in your mesh that you have permission to view.

**Review by JStickler — APPROVED** [review]

**Pull request merged**